scripts/time-drift.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# zenoss-inspector-tags verify
+#
+
+for i in $(serviced host list| awk '{print $4}' | grep -v Name | grep -v $HOSTNAME); do echo ntpdate \-q $i; ntpdate -q $i; done
```

Comment:
I'm not sure we can assume DNS entries for all hosts, so checking by IP might be more reliable.   Also, 
* You can simplify a little by asking just for the fields you want, thus removing the need for awk/
* I think a multiline impl would be a little easier for readability.
* If by chance the last host in the list is unreachable, then the script exits with a non-zero exit code and inspector will skip the execution of `check-time-drift.py` with an error message like "one or more of its dependencies are not available"

So I suggest something like this:

```
HOST_IP=`hostname -i`
for i in $(serviced host list --show-fields Addr | grep -v ^Addr  | grep -v $HOST_IP); do
   echo ntpdate \-q $i
   ntpdate -q $i
done
exit 0
```